### PR TITLE
DOC: Adds Iñigo's w10 and w11 blog posts 

### PR DESCRIPTION
**Pull request by itellaetxe**
Adds Iñigo's week 10 and week 11 blog posts.

Also adds necessary images to render properly.

I tried the inline LaTeX capabilities of RST in w11 blog post, should render fine (I believe).

### CODE REVIEW

**Review by WassCodeur — COMMENTED** [review]
Hi @itellaetxe!

Good work and nice clear articles. LGTM

**Review by robinroy03 — APPROVED** [review]

File: posts/2024/2024_08_09_Inigo_week_11.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+As for the AR, we try to tie a continuous attribute of choice found in the data space (fur length, age, size, etc.) to a specific dimension of the latent space. To do this, we compute an attribute-distance matrix in the data space :math:`D_a`, and we compute a distance matrix from the chosen dimension of the latent space (:math:`D_r`). By minimizing the mean absolute error (MAE) between the two matrices, we force the latent space to be organized in such a way that the chosen dimension is related to the chosen attribute. This way, we can generate samples conditioned on the attribute of choice. The AR loss (:math:`\mathcal{L}_{AR}`) is computed as it is shown in the top rectangle.
```
Comment (jhlegarreta) [inline posts/2024/2024_08_09_Inigo_week_11.rst:26]: Related to the above: (fur length, age, size, etc.)

**Review by jhlegarreta — COMMENTED** [review]
Thanks for this @itellaetxe.

A couple of inline comments.

File: posts/2024/2024_08_09_Inigo_week_11.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+In our case, the generator is the encoder :math:`\mathcal{G}`, which generates a latent representation of the input data, which the discriminator :math:`\mathcal{D}` tries to distinguish from "real" latent representations, sampled from a given prior distribution. The trick to introduce the information of the kind of animal to which the photo belongs to is to concatenate the latent representation with the one-hot encoded bundle and attribute vectors. This way, the decoder :math:`\mathcal{D}` can generate samples conditioned on a categorical variable. The reconstruction loss (:math:`\mathcal{L}_{MSE}`) is computed as it is shown in the middle rectangle, and it ensures that the samples are reconstructed from the latent representation as close as possible to the original data.
```
[the remark this answers is not in the document]
Comment (itellaetxe) [reply posts/2024/2024_08_09_Inigo_week_11.rst:24]: Oops, true. Will stick to animals to keep it simple.

File: posts/2024/2024_08_09_Inigo_week_11.rst (hunk 1 above)
Comment (itellaetxe) [reply posts/2024/2024_08_09_Inigo_week_11.rst]: Got it, thanks :)

**Review by skoudoro — APPROVED** [review]
Thank you for addressing the comments. All good, merging

**Pull request merged**